app/views/layouts/_navbar.html.erb

```diff
@@ -6,6 +6,7 @@
     </button>
     <div class="collapse navbar-collapse" id="navbarSupportedContent">
       <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+        <%= breadcrumbs display_single_fragment: true, separator: "&rsaquo;", style: :inline, class: "breadcrumb", link_class: "breadcrumb-link" %>
```

Comment:
Found one downside of putting these here - it's not visible anywhere on mobile. I think this is a good spot on bigger screens, but we may need a second instance of this somewhere that's only visible on mobile.